src/api/search/src/routes/query.js

```diff
@@ -1,22 +1,12 @@
 const { Router, createError } = require('@senecacdot/satellite');
 const { validateQuery } = require('../bin/validation');
-const { search, advancedSearch } = require('../search');
+const { search } = require('../search');
 
 const router = Router();
 
 router.get('/', validateQuery, async (req, res, next) => {
   try {
-    const { text, filter, page, perPage } = req.query;
```

Comment:
This is the MAIN part of the PR - removed the regular search to use the advanced route only(much more flexible and also covers all cases with the current search). I then cleaned up all the code from the regular search. So now there is no more `/advanced` ----> that route is now the main path `/`